src/simulators.jl

```diff
@@ -930,3 +930,54 @@ function random_unit_vector(float_type, dims)
     vec = randn(float_type, dims)
     return vec / norm(vec)
 end
+
+
+"""
+    OverdampedLangevin(; <keyword arguments>)
+
+Simulates the overdamped Langevin equation using the Euler-Maruyama method.
```

Comment:
Is there a canonical paper to reference here?

Reply:
The Euler Maruyama scheme is pretty standard, even Wikipedia forgoes the reference..
Considering the limit underdamped Langevin $\rightarrow$ overdamped Langevin and the resulting unit considerations I would point to the "Free energy computations" book by Lelievre, Rousset and Stoltz, but its merely hinted at there..